release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2310,6 +2310,8 @@ scheduling:
 
 * For clusters that use the OVN-Kubernetes network provider and whose compute nodes run RHEL 7.9, upgrading from {product-title} 4.7 to {product-title} 4.8 is blocked by link:https://bugzilla.redhat.com/show_bug.cgi?id=1976232[*BZ#1976232*]. To upgrade to release 4.8, you must wait for the 4.8 patch that includes the fix for this bug. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1976232[*BZ#1976232*])
 
+* For clusters that use the OVN-Kubernetes network provider and upgrade from {product-title} 4.7 to {product-title} 4.8, a bug in OVN-Kubernetes can sometimes cause the pod IP address to be recorded in OVN incorrectly. The bug is a rarely experienced race conditition. As a consequence, during the upgrade to the 4.8 release, nodes fail to drain and some Operators report a status of `Degraded`. As a workaround, identify the pods for the Operators that did not complete the upgrade and delete each pod with the `oc` command. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1974403[*BZ#1974403*])
```

Comment:
`a bug in OVN-Kubernetes can sometimes cause the pod IP address to be recorded in OVN incorrectly`-> `a bug in OVN-Kubernetes can sometimes cause the pod IP address to become stale`

Reply:
`As a workaround, identify the pods for the Operators that did not complete the upgrade and delete each pod with the `oc` command`  -> 
`As a workaround, identify pods stuck in `crashloopbackoff` that did not complete the upgrade and delete each pod with the `oc delete <podname>` command